framework/include/fvbcs/FVDirichletBC.h

```diff
@@ -9,12 +9,12 @@
 
 #pragma once
 
-#include "FVBoundaryCondition.h"
+#include "FVDirichletBCBase.h"
 
 /**
  * Base class for
  */
-class FVDirichletBC : public FVBoundaryCondition
+class FVDirichletBC : public FVDirichletBCBase
 {
 public:
   /**
```

Comment:
Github won't let me post a review comment on text below the diffs here - but throw in an `override` on the boundaryValue func for good measure.